tenders/common/mft.h

```diff
@@ -46,23 +46,28 @@ int mft_validate(struct mft *mft, size_t mft_size);
  * This is intended for use by bindings implementations that wish to access the
  * built-in copy of the application manifest in-place. You should call
  * mft_validate() on the returned values before using them.
+ *
+ * The two versions of this function provided differ only in the const-ness of
+ * the returned (*out_mft).
  */
-void mft_get_builtin_mft1(struct mft1_note *note, struct mft **out_mft,
-        size_t *out_mft_size);
+void mft_get_builtin_mft1_unconst(const struct mft1_note *note,
+        struct mft **out_mft, size_t *out_mft_size);
+void mft_get_builtin_mft1(const struct mft1_note *note,
+        const struct mft **out_mft, size_t *out_mft_size);
 
 /*
  * Return the manifest entry matching (name), of type (type), or NULL if none
  * found.
```

Comment:
Not related to this PR, but I think it would make sense if the comment documenting this function mentioned that if `index != NULL`, and if an entry is found, then the index will be stored to `*index` (so you can cache it and use `mft_get_by_index` in future calls I guess?)

Reply:
I'll update the comment.